[2024-01-31T12:26:48.721+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hola_mundo.print_hello_task scheduled__2024-01-17T00:00:00+00:00 [queued]>
[2024-01-31T12:26:49.047+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hola_mundo.print_hello_task scheduled__2024-01-17T00:00:00+00:00 [queued]>
[2024-01-31T12:26:49.048+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2024-01-31T12:26:49.608+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): print_hello_task> on 2024-01-17 00:00:00+00:00
[2024-01-31T12:26:49.645+0000] {standard_task_runner.py:60} INFO - Started process 96 to run task
[2024-01-31T12:26:49.673+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'hola_mundo', 'print_hello_task', 'scheduled__2024-01-17T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/prueba.py', '--cfg-path', '/tmp/tmp6cgn1yf3']
[2024-01-31T12:26:49.688+0000] {standard_task_runner.py:88} INFO - Job 19: Subtask print_hello_task
[2024-01-31T12:26:50.652+0000] {task_command.py:423} INFO - Running <TaskInstance: hola_mundo.print_hello_task scheduled__2024-01-17T00:00:00+00:00 [running]> on host 60382a02b079
[2024-01-31T12:26:51.884+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='usuario' AIRFLOW_CTX_DAG_ID='hola_mundo' AIRFLOW_CTX_TASK_ID='print_hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-17T00:00:00+00:00'
[2024-01-31T12:26:51.888+0000] {logging_mixin.py:188} INFO - Hola Mundo!
[2024-01-31T12:26:51.889+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-31T12:26:52.014+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=hola_mundo, task_id=print_hello_task, execution_date=20240117T000000, start_date=20240131T122648, end_date=20240131T122652
[2024-01-31T12:26:52.252+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T12:26:52.446+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
